mdsuite/calculators/green_kubo_distinct_diffusion_coefficients.py

```diff
@@ -232,7 +232,7 @@ def _post_operation_processes(self, species: Union[str, tuple] = None):
 
         properties = {"Property": self.database_group,
                       "Analysis": self.analysis_name,
-                      "Subject": species,
+                      "Subject": [species[0], species[1]],
```

Comment:
Did this really fail?

Reply:
I tried it with just the tuple and yes it failed.